evaluator.go

```diff
@@ -45,6 +45,11 @@ func Evaluate(tape string, out io.Writer, opts ...EvaluatorOption) error {
 		}
 	}
 
+	h, w, padding := v.Options.Video.Height, v.Options.Video.Width, v.Options.Video.Padding
+	if h < 2*padding || w < 2*padding {
+		return errors.New("height and width must be greater than " + fmt.Sprint(2*padding))
```

Comment:
In such situations `fmt.Errrorf` might be slightly preferable.

Reply:
Ah, great point. Will update